### PR TITLE
Remove Giphy tracks

### PR DESCRIPTION
**Pull request by ScoutHarris**
Ref: https://github.com/wordpress-mobile/WordPress-iOS/issues/13803
Can be tested with WPiOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/14302

This removes Giphy tracks as Giphy is no longer being used.

### CODE REVIEW

File: WordPressShared/Core/Analytics/WPAnalytics.h [hunk 1]
```diff
@@ -74,7 +74,6 @@ typedef NS_ENUM(NSUInteger, WPAnalyticsStat) {
     WPAnalyticsStatEditorAddedPhotoViaWPMediaLibrary,
     WPAnalyticsStatEditorAddedVideoViaWPMediaLibrary,
     WPAnalyticsStatEditorAddedOtherMediaViaWPMediaLibrary,
-    WPAnalyticsStatEditorAddedPhotoViaGiphy,
```
Comment (Gio2018) [inline WordPressShared/Core/Analytics/WPAnalytics.h:77]: Will the related Tenor tracks be added in a separate PR?

Comment (ScoutHarris) [reply WordPressShared/Core/Analytics/WPAnalytics.h:77]: Nope. The Tenor tracks are using the [new system](https://wp.me/paNNhX-55) that doesn't require Shared. And have [already been added](https://github.com/wordpress-mobile/WordPress-iOS/pull/13923) to WP.
